<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>febDialog</class>
 <widget class="QDialog" name="febDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>FEB Command Generator</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>10</y>
         <width>171</width>
         <height>151</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Insert Analog Value</string>
       </property>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>141</width>
          <height>24</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Convert</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>30</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>DAC Value</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_dacValue">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>50</y>
          <width>61</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>50</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>mV</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>581</width>
         <height>151</height>
        </rect>
       </property>
       <property name="title">
        <string>Select Parameter</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>32</y>
          <width>74</width>
          <height>52</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="rB_Feb_Read">
           <property name="text">
            <string>Read FEB</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rB_Feb_Set">
           <property name="text">
            <string>Wrire FEB</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>32</y>
          <width>135</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="rB_DB_No1">
           <property name="text">
            <string>Distribution Board No.1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rB_DB_No2">
           <property name="text">
            <string>Distribution Board No.2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rB_DB_No3">
           <property name="text">
            <string>Distribution Board No.2</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>32</y>
          <width>143</width>
          <height>52</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QRadioButton" name="rB_DB_P1">
           <property name="text">
            <string>Distribution Board Port 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rB_DB_P2">
           <property name="text">
            <string>Distribution Board Port 2</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>32</y>
          <width>67</width>
          <height>108</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QRadioButton" name="rB_Feb1">
           <property name="text">
            <string>FEB No1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rB_Feb2">
           <property name="text">
            <string>FEB No2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rB_Feb3">
           <property name="text">
            <string>FEB No3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rB_Feb4">
           <property name="text">
            <string>FEB No4</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>32</y>
          <width>55</width>
          <height>108</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QRadioButton" name="rB_Dac1">
           <property name="text">
            <string>DAC 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rB_Dac2">
           <property name="text">
            <string>DAC 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rB_Adc1">
           <property name="text">
            <string>ADC1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rB_Adc2">
           <property name="text">
            <string>ADC 2</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>32</y>
          <width>60</width>
          <height>108</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QRadioButton" name="rB_Vth1">
           <property name="text">
            <string>Vth1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rB_Vth2">
           <property name="text">
            <string>Vth2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rB_Vmon1">
           <property name="text">
            <string>Vmon1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rB_Vmon2">
           <property name="text">
            <string>Vmon2</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>31</y>
          <width>16</width>
          <height>81</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="Line" name="line_2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>31</y>
          <width>16</width>
          <height>81</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="Line" name="line_3">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>31</y>
          <width>16</width>
          <height>111</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="Line" name="line_4">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>30</y>
          <width>16</width>
          <height>111</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="Line" name="line_5">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>31</y>
          <width>16</width>
          <height>111</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>180</y>
         <width>151</width>
         <height>71</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>160</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>128 Bit Result (HEX) </string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_2">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>180</y>
         <width>281</width>
         <height>71</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>160</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>128 Bit Result (BIN) </string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>180</y>
         <width>21</width>
         <height>71</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>D(3)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>D(2)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>D(1)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>D(0)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>180</y>
         <width>21</width>
         <height>71</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>D(3)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>D(2)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>D(1)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>D(0)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Adjusts</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New </string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Column</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
